system-tests/projects/firefox-memory/cypress/e2e/spec.js

```diff
@@ -50,12 +50,8 @@ describe('memory leak finder', function () {
   after(() => {
     duration = Date.now() - duration
 
-    cy
-    .task('stop:capture:memory')
-    .then(() => {
-      return Cypress.backend('log:memory:pressure')
-    })
-    .task('console', { duration })
+    cy.task('stop:capture:memory')
```

Comment:
do we need this system test anymore? currently the `firefox-memory` project is only being used in the `firefox_spec`, and the test looks to be indefinitely skipped https://github.com/cypress-io/cypress/blob/remove-ff-gc/system-tests/test/firefox_spec.ts#L18

Reply:
Ah, good catch. Let me look.

Reply:
@AtofStryker Removed in [`1d3bea0` (#30331)](https://github.com/cypress-io/cypress/pull/30331/commits/1d3bea0a0f5a063f8782989eac5fe9ca2fb31ce3)